src/visual.ts

```diff
@@ -987,7 +987,7 @@ module powerbi.extensibility.visual {
                 .x((dataPoint: StreamDataPoint) => xScale(dataPoint.x))
                 .y0((dataPoint: StreamDataPoint) => yScale(dataPoint.y0))
                 .y1((dataPoint: StreamDataPoint) => yScale(dataPoint.y0 + dataPoint.y))
-                .defined((dataPoint: StreamDataPoint) => !isNaN(dataPoint.y0) && !isNaN(dataPoint.y));
+                .defined((dataPoint: StreamDataPoint) => !isNaN(dataPoint.y0) && dataPoint.y0 !== null && !isNaN(dataPoint.y) && dataPoint.y !== null);
```

Comment:
Please prepare a Unit Test that covers this issue to exclude any further regressions.